### DISCUSSION TITLE
Bug report: can't apply font size to entire note

### DISCUSSION THREAD

**Discussion by semidivine** [standardnotes/forum#2795] [General]
I am using the plus editor on Standard Notes v. 3.14.1 on a Mac running OS Catalina 10.15.7.

I am experiencing a frustrating bug where I select an entire note of text (in this case two headers and two unordered lists) and when I select the text and apply a font-size change it only changes for the first element. So first time the only text that increased font size was the first header. Then I selected the entire first ordered list after the header and it only changed font size for the first list item.

The result is I have to highlight and change font size on each line of the unordered lists one list item at a time. This is verging on a deal breaker for me using this app because I am bringing in tons of notes one at a time from Mac Notes and this bug will make this process that much harder.

I've attached a screen capture of what I am experiencing.

https://user-images.githubusercontent.com/6435588/165868572-e44fbcd7-7b5f-447c-88ae-3f317afeeffc.mov

PS. the Chicken pot pie recipe is great.

**Comment by user4-33** [comment]
Having the same issue, takes much energy...
